### PR TITLE
4.x: Docs: Fix SE Health endpoint path to match code snippet

### LINKED ISSUE #8485: 4.x: docs se/health k8s example snippet wrong annotations
## Environment Details
* Helidon Version: 4.0.5
* Helidon SE

----------

## Problem Description

The snippets shows how to set the endpoints as "/health/xxx" but the annotation describe the code as "/observer/health/xxx".

See:
- https://helidon.io/docs/4.0.5/se/health#_kubernetes_example
- https://github.com/helidon-io/helidon/blob/4.0.6/docs/src/main/java/io/helidon/docs/se/HealthSnippets.java#L157
- https://github.com/helidon-io/helidon/blob/4.0.6/docs/src/main/asciidoc/se/health.adoc?plain=1#L495
- https://github.com/helidon-io/helidon/blob/4.0.6/docs/src/main/java/io/helidon/docs/se/HealthSnippets.java#L162
- https://github.com/helidon-io/helidon/blob/4.0.6/docs/src/main/asciidoc/se/health.adoc?plain=1#L497

### PR DESCRIPTION
**Pull request by barchetta**
### Description

Corrects SE health endpoint path in text to match example code snippet.

Fixes #8485

### Documentation

Fixes documentation

### CODE REVIEW

**Review by tjquinno — APPROVED** [review]

**Comment by barchetta** [comment]
Ooops! I think the bug wants the fix the other way around (leave snippet, change text)

File: docs/src/main/asciidoc/se/health.adoc [hunk 1]
```diff
@@ -492,9 +492,9 @@ specification is also provided for the Kubernetes service and deployment.
 ----
 include::{sourcedir}/se/HealthSnippets.java[tag=snippet_8, indent=0]
 ----
-<1> The health service for the `liveness` probe is exposed at `/observe/health/live`.
+<1> The health service for the `liveness` probe is exposed at `/health/live`.
 <2> Using the built-in health checks for the `liveness` probe.
-<3> The health service for the `readiness` probe is exposed at `/observe/health/ready`.
+<3> The health service for the `readiness` probe is exposed at `/health/ready`.
```
Comment (tjquinno) [inline docs/src/main/asciidoc/se/health.adoc:497]: If this is SE with no config changes to alter the default endpoint paths, then the `/observe` prefix _should_ be present.

**Review by tjquinno — CHANGES_REQUESTED** [review]
Why remove `/observe`? That's the default, unless changed.

File: docs/src/main/asciidoc/se/health.adoc (hunk 1 above)
Comment (barchetta) [reply docs/src/main/asciidoc/se/health.adoc:497]: See comment above. This text is annotating a code snippet that changes the default endpoint path to "/health". The bug is that the text annotations did not match the code snippet.

Comment (tjquinno) [reply docs/src/main/asciidoc/se/health.adoc:497]: Ugh. Sorry.

**Review by tjquinno — APPROVED** [review]

**Pull request merged**